dg/media/app_dashboards/js/loop_dashboard.js

```
@@ -2876,35 +2969,382 @@ function aggregator_payment_sheet(data_json, aggregator) {
+            $this.parent()[0].childNodes[4].innerHTML = $('#aggregator_share_row').val();
+            $this.parent()[0].childNodes[9].innerHTML = $('#aggregator_comment_row').val();
+            $this.parent()[0].childNodes[8].innerHTML = parseFloat(parseFloat($this.parent()[0].childNodes[5].innerHTML) + parseFloat($this.parent()[0].childNodes[4].innerHTML) - parseFloat($this.parent()[0].childNodes[7].innerHTML) - parseFloat($this.parent()[0].childNodes[6].innerHTML)).toFixed(2)
+            if (parseFloat($this.parent()[0].childNodes[4].innerHTML / $this.parent()[0].childNodes[3].innerHTML.split('Kg')[0]) > 0.5) {
+                //$this.parent().css('background-color', '#E5FEB5').css('font-weight', 'bold').css('color', '#009');
+                if(editedAggregator==1)
+                    $this.css('background-color', '#F8C6B8').css('font-weight', 'bold').css('color', '#009');
+                else{
+                    $this.parent()[0].childNodes[9].style.backgroundColor= '#E5FEB5';
+                    $this.parent()[0].childNodes[9].style.fontWeight ='bold';
+                    $this.parent()[0].childNodes[9].style.color = '#009';
+                }
+            } else {
+                if(editedAggregator==1)
+                    $this.css('background-color', '#E5FEB5').css('font-weight', 'bold').css('color', '#009');
+                //$this.parent().css('background-color', '#E5FEB5').css('font-weight', 'bold').css('color', '#009');
+                else{
+                    $this.parent()[0].childNodes[9].style.backgroundColor= '#E5FEB5';
+                    $this.parent()[0].childNodes[9].style.fontWeight ='bold';
+                    $this.parent()[0].childNodes[9].style.color = '#009';
+                }
+            }
+            var row_id = $this.context.parentNode.rowIndex;
+            rows_table2[row_id] = true;
+            editedAggregator =0;
+            $('#aggregator_error_div').hide();
```

No need to hide the error div. Already hidden when initialized.